javalanglib/src/main/scala/java/lang/Character.scala

```diff
@@ -519,6 +518,971 @@ object Character {
+      OLD_ITALIC,
+      GOTHIC,
+      OLD_PERMIC,
+      UGARITIC,
+      OLD_PERSIAN,
+      DESERET,
+      SHAVIAN,
+      OSMANYA,
+      OSAGE,
+      ELBASAN,
+      CAUCASIAN_ALBANIAN,
+      LINEAR_A,
+      CYPRIOT_SYLLABARY,
+      IMPERIAL_ARAMAIC,
+      PALMYRENE,
+      NABATAEAN,
+      HATRAN,
+      PHOENICIAN,
+      LYDIAN,
+      MEROITIC_HIEROGLYPHS,
+      MEROITIC_CURSIVE,
+      KHAROSHTHI,
+      OLD_SOUTH_ARABIAN,
+      OLD_NORTH_ARABIAN,
+      MANICHAEAN,
+      AVESTAN,
+      INSCRIPTIONAL_PARTHIAN,
+      INSCRIPTIONAL_PAHLAVI,
+      PSALTER_PAHLAVI,
+      OLD_TURKIC,
+      OLD_HUNGARIAN,
+      HANIFI_ROHINGYA,
+      RUMI_NUMERAL_SYMBOLS,
+      OLD_SOGDIAN,
+      SOGDIAN,
+      ELYMAIC,
+      BRAHMI,
+      KAITHI,
+      SORA_SOMPENG,
+      CHAKMA,
+      MAHAJANI,
+      SHARADA,
+      SINHALA_ARCHAIC_NUMBERS,
+      KHOJKI,
+      MULTANI,
+      KHUDAWADI,
+      GRANTHA,
+      NEWA,
+      TIRHUTA,
+      SIDDHAM,
+      MODI,
+      MONGOLIAN_SUPPLEMENT,
+      TAKRI,
+      AHOM,
+      DOGRA,
+      WARANG_CITI,
+      NANDINAGARI,
+      ZANABAZAR_SQUARE,
+      SOYOMBO,
+      PAU_CIN_HAU,
+      BHAIKSUKI,
+      MARCHEN,
+      MASARAM_GONDI,
+      GUNJALA_GONDI,
+      MAKASAR,
+      TAMIL_SUPPLEMENT,
+      CUNEIFORM,
+      CUNEIFORM_NUMBERS_AND_PUNCTUATION,
+      EARLY_DYNASTIC_CUNEIFORM,
+      EGYPTIAN_HIEROGLYPHS,
+      EGYPTIAN_HIEROGLYPH_FORMAT_CONTROLS,
+      BAMUM_SUPPLEMENT,
+      MRO,
+      BASSA_VAH,
+      PAHAWH_HMONG,
+      MEDEFAIDRIN,
+      MIAO,
+      TANGUT,
+      TANGUT_COMPONENTS,
+      KANA_SUPPLEMENT,
+      KANA_EXTENDED_A,
+      SMALL_KANA_EXTENSION,
+      NUSHU,
+      DUPLOYAN,
+      SHORTHAND_FORMAT_CONTROLS,
+      BYZANTINE_MUSICAL_SYMBOLS,
+      MUSICAL_SYMBOLS,
+      ANCIENT_GREEK_MUSICAL_NOTATION,
+      MAYAN_NUMERALS,
+      TAI_XUAN_JING_SYMBOLS,
+      COUNTING_ROD_NUMERALS,
+      MATHEMATICAL_ALPHANUMERIC_SYMBOLS,
+      GLAGOLITIC_SUPPLEMENT,
+      NYIAKENG_PUACHUE_HMONG,
+      WANCHO,
+      MENDE_KIKAKUI,
+      ADLAM,
+      INDIC_SIYAQ_NUMBERS,
+      OTTOMAN_SIYAQ_NUMBERS,
+      ARABIC_MATHEMATICAL_ALPHABETIC_SYMBOLS,
+      MAHJONG_TILES,
+      DOMINO_TILES,
+      PLAYING_CARDS,
+      ENCLOSED_ALPHANUMERIC_SUPPLEMENT,
+      ENCLOSED_IDEOGRAPHIC_SUPPLEMENT,
+      EMOTICONS,
+      ORNAMENTAL_DINGBATS,
+      ANATOLIAN_HIEROGLYPHS,
+      GEOMETRIC_SHAPES_EXTENDED,
+      SUPPLEMENTAL_ARROWS_C,
+      CHESS_SYMBOLS,
+      CJK_UNIFIED_IDEOGRAPHS_EXTENSION_B,
+      CJK_UNIFIED_IDEOGRAPHS_EXTENSION_C,
+      CJK_UNIFIED_IDEOGRAPHS_EXTENSION_D,
+      CJK_UNIFIED_IDEOGRAPHS_EXTENSION_E,
+      CJK_UNIFIED_IDEOGRAPHS_EXTENSION_F,
+      CJK_COMPATIBILITY_IDEOGRAPHS_SUPPLEMENT,
+      TAGS,
+      VARIATION_SELECTORS_SUPPLEMENT,
+      SUPPLEMENTARY_PRIVATE_USE_AREA_A,
+      SUPPLEMENTARY_PRIVATE_USE_AREA_B
+    )
+
+    private val blocksByNormalizedName: java.util.Map[String,UnicodeBlock] = {
```

Comment:
This map could also be built at run-time through side-effects in the constructors of `UnicodeBlock`.